web/src/components/TriggerTypeModal/TriggerTypeModal.tsx

```diff
@@ -0,0 +1,45 @@
+import {TriggerTypeToPlugin} from 'constants/Plugins.constants';
+import {useDashboard} from 'providers/Dashboard/Dashboard.provider';
+import CreateTestAnalyticsService from 'services/Analytics/CreateTestAnalytics.service';
+import TriggerTypeCard from './TriggerTypeCard';
+import * as S from './TriggerTypeModal.styled';
+
+const pluginList = Object.values(TriggerTypeToPlugin);
+
+interface IProps {
+  isOpen: boolean;
+  onClose(): void;
+}
+
+const TriggerTypeModal = ({isOpen, onClose}: IProps) => {
+  const {navigate} = useDashboard();
+
+  return (
+    <S.Modal
+      onCancel={onClose}
+      footer={null}
+      title={<S.Title level={2}>Create a new test</S.Title>}
+      visible={isOpen}
+      width={625}
+    >
+      <S.Title level={3} $marginBottom={16}>
+        What kind of trigger do you want to use to initiate this Tracetest?
+      </S.Title>
+
+      <S.CardList>
```

Comment:
I don't see the coming soon cards, is this what we want?

Reply:
We only have the `OpenAPI` as coming soon and I think that is going to be an import, not a create flow. Anyway we can add it if we want